src/main/java/datawave/data/normalizer/NumericRegexNormalizer.java

```diff
@@ -0,0 +1,706 @@
+package datawave.data.normalizer;
+
+import com.google.common.base.CharMatcher;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+import datawave.data.type.util.NumericalEncoder;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+import java.util.stream.Collectors;
+
+public class NumericRegexNormalizer {
```

Comment:
Did you evaluate the use of the JavaRegexAnalyzer in the process of doing this ticket?  Just curious.

Reply:
I didn't know the JavaRegexAnalyzer existed when I completed this work, but I will evaluate it.